### ISSUE TITLE
Automate themeing function documentation in readme file

### ISSUE THREAD

**Issue by fabacab** [meitar/multisite-directory#6]
See 55507b052c5656c636cb28c5d305937747557bac for the expected input/output, this should be automated and not a copy-paste-edit manual operation.

**Comment by fabacab** [comment]
This is actually a general problem that should be solved for all plugins, so I'm gonna write [a phpDoc template to do this](https://github.com/meitar/phpdoc-wp-readme.txt). Progress will be tracked there.

**Comment by misfist** [comment]
Yes, that would be great! I forget to do it all the time! 

Also, can you recommend a plugin boilerplate that isn't totally overblown? I've tried https://github.com/DevinVinson/WordPress-Plugin-Boilerplate, but find it too much for most of my needs.

**Comment by fabacab** [comment]
I've never made a phpDoc template before so [I could use some help figuring out how to do it](https://joindiaspora.com/posts/7185853). Especially considering [phpDocumentor's documentation, ironically, sucks](https://joindiaspora.com/posts/7185193).

As for a boilerplate, no, I don't like any. I always get the feeling that their creators are like, "this project is for a boilerplate, so what it really needs is more boilerplate," and I just do a headdesk cuz, augh, that is the opposite of the point.

I tend to use [my own skeleton code](https://github.com/meitar/wordpress-plugin-skeleton) as a starting point a lot, which is similar to [WP-CLI's `scaffold plugin`](http://wp-cli.org/commands/scaffold/plugin/) and that's often more than enough for me to get started.

**Comment by misfist** [comment]
I've never done it either :/ They make it look easy here: https://phpdoc.org/docs/latest/getting-started/your-first-set-of-documentation.html#running-phpdocumentor 

Thanks for the skeleton link, I'll check it out.

**Comment by fabacab** [comment]
Yeah, those are docs about how to generate docs, not how to customize the output template. If you follow through the links you'll find a lot of dead ends. I resorted to reading the templates phpDoc ships with and trying to poke it in enough different ways to do what I want. I was able to get as far as correctly looping over the functions defines in a single file, but that's all. Anyway this should probably be discussed over at the other repo's issue queue. Any help would be appreciated but don't stress, I know you have a day job and I don't have that burden.
